lib/eventbrite/util.rb

```diff
@@ -60,7 +60,7 @@ def self.symbolize_names(object)
     end
 
     def self.url_encode(key)
-      URI.escape(key.to_s, Regexp.new("[^#{URI::PATTERN::UNRESERVED}]"))
+      CGI.escape(key.to_s)
```

Comment:
This method isn't exactly the same, but it's suggested as a replacement. Hopefully it'll work fine in this case.